README.md

```diff
@@ -50,36 +56,37 @@ This is a two part process:
 Run the following command to publish all schemas to the Iglu server bundled with Snowplow-mini:
 
 ```
-$ ./iglu_server_upload.sh http://{{ snowplow-mini ip }}:8081 {{ snowplow-mini iglu server key (uuid) }} schemas
+$ /path/to/igluctl static push ./schemas {{ snowplow-mini-ip}}:8081 {{ snowplow-mini-iglu-master-key }} --public
 ```
 
-
 Note that you can specify individual schemas if you prefer e.g.
 
 ```
-$ ./iglu_server_upload.sh http://{{ snowplow-mini ip }}:8081 {{ snowplow-mini iglu server key (uuid) }} schemas/com.mycompany/my_new_event_schema 
+$ /path/to/igluctl static push ./schemas/com.mycompany/my_new_event_schema {{ snowplow-mini-ip}}:8081 {{ snowplow-mini-iglu-master-key }} --public
```

Comment:
Rather than inventing a pseudocode template language, I prefer to use Bash variables. So you would introduce the variables you need once:

    $ snowplow_mini_ip=212.32.33.14
    $ access_key_id=ABCD

and then you would just reference these directly:

    $ /path/to/igluctl static push ./schemas/com.mycompany/my_new_event_schema ${snowplow_mini_ip}:8081 ${snowplow_mini_iglu_master_key} --public

This has two advantages:

* It forces you to define your terms (when you declare the variables)
* It means the actual code lines can be copy-pasted direct into a terminal without editing

You could also add the Windows version alongside (http://steve-jansen.github.io/guides/windows-batch-scripting/part-2-variables.html)